source/color_raw_to_rgb.ipynb

```diff
@@ -0,0 +1,328 @@
+{
```

Comment:
<p>I think it will fail. tutorials are a bit tricky here: there must be only a title with # , the rest must go under with ##, ###m, etc</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kornia/tutorials/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Looks fine in colab, will split just in case.</p>